src/presentation/cells/autocompletion.ts

```diff
@@ -41,7 +41,7 @@ export function alternativesProviderForAddingChild(
         }
       };
       const uiAlternatives = Array.from(
-        $(alternatives).map((index, domElement: Alternative) => {
+        alternatives.map((domElement, index) => {
```

Comment:
are domElement and index inverted on purpose?

Reply:
Yes, the signatures of the jQuery map and the JavaScript map are different.